sequencer.c

```diff
@@ -2653,6 +2653,33 @@ int sequencer_rollback(struct repository *r, struct replay_opts *opts)
 	return -1;
 }
 
+int sequencer_skip(struct repository *r, struct replay_opts *opts)
+{
+	if (!file_exists(git_path_cherry_pick_head(r)) &&
+	    !file_exists(git_path_revert_head(r)))
+		return error(_("no cherry-pick or revert in progress"));
+
+	struct child_process reset = CHILD_PROCESS_INIT;
+	reset.git_cmd = 1;
+
+	argv_array_push(&reset.args, "reset");
+	argv_array_push(&reset.args, "--merge");
+
+	run_command(&reset);
+
+	if (file_exists(git_path_seq_dir())) {
+		struct child_process cherry_pick = CHILD_PROCESS_INIT;
+		cherry_pick.git_cmd = 1;
+
+		argv_array_push(&cherry_pick.args, "cherry-pick");
+		argv_array_push(&cherry_pick.args, "--continue");
```

Comment:
Isn't there also a revert --continue, for when someone tries to revert a sequence of commits?